ui/src/common/components/CitationsByYearGraph/CitationsByYearGraph.jsx

```diff
@@ -16,6 +16,7 @@ import pluralizeUnlessSingle, {
 import EmptyOrChildren from '../EmptyOrChildren';
 
 const BLUE = styleVariables['@primary-color'];
+const LIGHT_BLUE = '#1890FF99';
```

Comment:
isn't it the primary with opacity? 🤔

Reply:
no no I'm just saying if we can have a styleVar (@primary with opacity) and use it here instead of hardcoding it